### ISSUE TITLE
[SI-ERROR-TRUE] 1.4.6 Contrast (Enhanced) - Color contrast is not sufficient

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#947]
## Problem Description per Siteimprove

The color of the text and the color of the background are not in sufficient contrast to each other. The contrast ratio should be at least 7:1 for normal text and 4.5:1 for large text. Note: Contrasts are not relevant where content is hidden. But make sure that for hidden content, which becomes visible when receiving focus, the contrast ratio is sufficient.

## Page Description

Any page with body content that contains links, including 'Skip to Content' and the breadcrumb trail. 

## What we're doing: 

We're using CSS to change the default blue to #006db1 which registers a contrast ration of 5.49 out of 7. 

## Solution

Punch up that blue a little. WebAIM.org suggests 'For example, if your page has black text on a white background and you use the standard blue color for links, the link color must be between approximately this color of blue (#6e5eff) and this color of blue (#531fff).'
